internal/obsreportconfig/obsreportconfig.go

```diff
@@ -88,6 +88,14 @@ func allViews() *ObsMetrics {
 	tagKeys = []tag.Key{obsmetrics.TagKeyExporter}
 	views = append(views, genViews(measures, tagKeys, view.Sum())...)
 
+	errorNumberView := &view.View{
+		Name:        "ExporterFailedNumber",
```

Comment:
Replace "ExporterFailedNumber" with "exporter/send_failed_requests"? To be consistent with other views.

Reply:
good point, will change to that